### DISCUSSION TITLE
CI/CD for react-email packages

### DISCUSSION THREAD

**Discussion by JorbFreire** [resend/react-email#646] [Ideas]
### Goals

1. Speed up new releases.

### Background

It may be not a feature request but looks like it.

### Proposal

Always publish a new version once a pull request is merged into main.
adding a new version upgrading on end like:
v0.6.0 -> v0.6.1 -> v0.6.2 -> v0.6.3
Bigger version updates could be done manually. or with another rule.
I would do it with github actions, but I'm not sure if it's desirable or if GitHub actions is the right tool for it.

**Comment by KarloDerEchte** [comment]
Yes this would be great!

**Comment by bukinoshita** [comment]
Would love to have something like this, how can we do it and what should we test? 

Few small things

- Check PR title
- Lint
- Build
- Format
